code/game/machinery/computer/dna_console.dm

```diff
@@ -126,9 +126,6 @@
 	if(user.client)
 		var/datum/asset/simple/assets =  get_asset_datum(/datum/asset/simple/genetics)
 		assets.send(user.client)
-	if(!(in_range(src, user) || issilicon(user)))
```

Comment:
without this check people can use this UI from anywhere within visual range

Reply:
I test it and people can't use this UI from anywhere within visual range.
The real check now in another place.